### ISSUE TITLE
Add $paths variable

### ISSUE THREAD

**Issue by adrianbj** [processwire/processwire-requests#127]
### Short description of the enhancement

Since we now have `$urls`, I find myself going to use `$paths`

I think it would be a nice addition for the sake of API consistency.

**Comment by teppokoivula** [comment]
+1 from the point of API consistency, but personally I'm guilty of using both $urls and $paths in my own code. Just wanted to point out that the more global variables we introduce, more common it will become that users inadvertently override them.

I'm definitely a fan of having just one $wire variable or wire() function for accessing all PW stuff :)

**Comment by adrianbj** [comment]
Good point @teppokoivula - I think I would actually prefer that too. I have used both on many occasions as well. I have used $files as well. $urls and $files are relatively recent additions to PW but I guess there is no going back now :)
